### PR TITLE
Uplink TC Rebalance

### PR DESCRIPTION
**Pull request by therealDLondon**
## About the PR
Many, many uplink items have had their TC cost adjusted during admin discussion

## Why / Balance
To promote the use of less-used items and discourage playstyles and loadouts that dont provide any meaningful RP.
Emag, Syndicate Bomb, Thieving Gloves, were all too powerful for their TC cost, ontop of the Emag being unironicaly the most purchased item.

## Technical details
Changed the uplink catalogue

I also with close to 100% certainty fucked some syntax up **PLEASE YELL AT ME IF I DID**

## Media
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: therealDLondon, Lyndomen, and Colin_Tel
- tweak: Fluctuations in the Syndicate Stock Market have changed the prices of various uplink items. Most cheaper, some more expensive.

### CODE REVIEW

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -311,7 +311,7 @@
   description: uplink-exploding-syndicate-bomb-desc
   productEntity: SyndicateBomb
   cost:
-    Telecrystal: 11
+    Telecrystal: 14 # DV - was 11, Syndicate bomb was way too powerful for the 11 TC cost
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:314]: not really sure what this does. the Syndie Bomb is already 11 TC which locks you out of any meaningful Syndie loadouts already, this just further locks you out of any meaningful loadouts but doesn't address the issue that this is `too strong`

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 2]
```diff
@@ -1004,7 +1013,7 @@
     Telecrystal: 6
   categories:
     - UplinkAllies
-  
+
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1016]: :(

**Review by DangerRevolution — COMMENTED** [review]
very cool

**Comment by DangerRevolution** [comment]
> not really sure what this does. the Syndie Bomb is already 11 TC which locks you out of any meaningful Syndie loadouts already, this just further locks you out of any meaningful loadouts but doesn't address the issue that this is `too strong`

To just elaborate a little more; if I want to bomb the hell out of a department, I can buy this knowing I'm committing to bombing the hell out of a department and I lock myself out of any useful purchases.

With this change to `14 TC`, I can buy this knowing I'm committing to bombing the hell out of a department and I lock myself out of any useful purchases.

**Review by Axionyxx — APPROVED** [review]

**Pull request merged**